server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -26,6 +26,10 @@ See the radio-option below as an example.
   margin: 0 auto;
 }
 
+.usa-alert .usa-alert__heading {
```

Comment:
Non-blocking - @ssandbekkhaug - Can you add a before and after pic of your changes in the PR description? I honestly cannot tell the difference between the screens on the Screenshot test.

Reply:
nm- I see the difference.